SCClassLibrary/Common/Control/Buffer.sc

```diff
@@ -494,47 +494,19 @@ Buffer {
 	}
 
 	sine1 { arg amps, normalize=true, asWavetable=true, clearFirst=true;
-		if(bufnum.isNil) { Error("Cannot call % on a % that has been freed".format(thisMethod.name, this.class.name)).throw };
-		server.listSendMsg(
-			[\b_gen, bufnum, "sine1",
-				normalize.binaryValue
-				+ (asWavetable.binaryValue * 2)
-				+ (clearFirst.binaryValue * 4)]
-				++ amps
-			)
+		server.listSendMsg(this.sine1Msg(amps, normalize, asWavetable, clearFirst))
```

Comment:
error messages for these don't appear to be updated, is that intentional?

Reply:
'write', 'copy' etc. are verbs.

'sine1' is not.

"You cannot 'sine1' a buffer that has already been freed" -- that sort of verbiage would not be an improvement.

Reply:
thanks, just making sure